### ISSUE TITLE
Need updates for performance claims in the docs

### ISSUE THREAD

**Issue by Galaxy0419** [tiangolo/fastapi#5259]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
N/A
```

### Description

The [link](https://github.com/tiangolo/fastapi/blob/master/docs/en/docs/index.md) used in the documentation refers a continues run benchmark of TechEmpower.  According to the website, the result is unofficial and incomplete:

> You are viewing results from run-id [7464e520-0dc2-473d-bd34-dbdfd7e85911](https://tfb-status.techempower.com/results/7464e520-0dc2-473d-bd34-dbdfd7e85911). These results are intended for review by project participants. Regular visitors are encouraged to view an official round using the navigation bar above.

Please update the link to use [round 21 scores](https://www.techempower.com/benchmarks/#section=data-r21&test=query) as a performance reference.

Meanwhile, the documentation above also claims the performance of FastAPI is "on par with NodeJS and Go".  FastAPI ranks at 202nd place according to the TechEmpower round 21 benchmark.  FastAPI is faster than nodejs as it ranks at 311.  However, the claim is not valid for go frameworks.  The popular go framework [fasthttp](https://github.com/valyala/fasthttp) ranks at 75th place.  Not even mentioning the fastest go implementation using fiber-prefork, which ranks at 53rd place.  FastAPI's performance is 20% slower than fiber-prefork.  It is NOT "on par with" the go frameworks.

### Operating System

Other

### FastAPI Version

N/A

### Python Version

N/A

**Comment by v3ss0n** [comment]
That would decrease stars.

**Comment by Galaxy0419** [comment]
> That would decrease stars.

I think honesty is more important than stars.  Especially the performance claims for projects have "fast" in its name.  Nobody would expect an interpreted language is faster than a compiled language anyway.  Being faster than nodejs is already a huge win IMO.  It's bad when people start doing their own benchmarks only to find out that the framework is not as performant as they thought.

**Comment by v3ss0n** [comment]
its important for the owner , not me or you.

**Comment by Galaxy0419** [comment]
> its important for the owner , not me or you.

Yes, starts are important, but do you think the owner want to make people think he is a biased person that use unofficial data for his project?
